WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Plugins/PluginDetailsViewModelTests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
+@testable import WooCommerce
+
+final class PluginDetailsViewModelTests: XCTestCase {
```

Comment:
These tests are a bit redundant, and can be removed if you think they should. I mainly added them because the class had no tests at all, so we have something that fails if needed. I believe we have no tests here because the way we're creating the viewmodel (eg: plugin is settable) is not quite the best. Ideally we can do some refactoring in the future, but it was a bit out of scope for this HACK project.